modules/Core/pages/register.php

```diff
@@ -23,18 +23,13 @@
 require_once(ROOT_PATH . '/modules/Core/includes/emails/register.php');
 
 // Check if registration is enabled
-$registration_enabled = DB::getInstance()->get('settings', ['name', 'registration_enabled'])->results();
-$registration_enabled = $registration_enabled[0]->value;
+$registration_enabled = Util::getSetting('registration_enabled');
 
 if ($registration_enabled == 0) {
     // Registration is disabled, display a message
     // Get registration disabled message and assign to Smarty variable
-    $registration_disabled_message = DB::getInstance()->get('settings', ['name', 'registration_disabled_message'])->results();
-    if (count($registration_disabled_message)) {
-        $message = Output::getPurified($registration_disabled_message[0]->value);
-    } else {
-        $message = 'Registration is currently disabled.';
-    }
+    $fallback_message = 'Registration is currently disabled.';
+    $message = Output::getPurified(Util::getSetting('registration_disabled_message', $fallback_message));
```

Comment:
```suggestion
    $message = Output::getPurified(Util::getSetting('registration_disabled_message', 'Registration is currently disabled.'));
```

Reply:
Decided to make it translatable instead :D